modules/appeals_api/app/workers/appeals_api/daily_stuck_records_report.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'sidekiq/monitored_worker'
+
+class AppealsApi::DailyStuckRecordsReport
+  include Sidekiq::Worker
+  include Sidekiq::MonitoredWorker
+
+  # Only retry for ~8 hours since the job runs daily
+  sidekiq_options retry: 11, unique_for: 8.hours
+
+  def perform
+    return unless enabled?
+
+    params = find_stuck_record_data(AppealsApi::HigherLevelReview) + \
+             find_stuck_record_data(AppealsApi::NoticeOfDisagreement) + \
+             find_stuck_record_data(AppealsApi::SupplementalClaim)
+
+    return if params.empty?
+
+    AppealsApi::Slack::Messager.new(params, notification_type: :stuck_record).notify!
+  end
+
+  def enabled?
+    Flipper.enabled? :appeals_daily_stuck_records_report
```

Comment:
Let's prefix with `decision_reviews` instead of `appeals` to match the bulk of our other Flipper settings